extensions/wikia/PageHeader/templates/PageHeaderController_index.php

```diff
@@ -1,9 +1,16 @@
 <header id="PageHeader" class="page-header">
-	<?= $app->renderView( 'Wikia\PageHeader\PageHeaderController', 'categories' ) ?>
-	<h1 class="page-header__title"><?= !empty( $pageTitle->prefix ) ? '<span>' . $pageTitle->prefix . ':</span> ' : '' ?><?= $pageTitle->title ?></h1>
-	<? if ( $counter->isNotEmpty() ) : ?>
-		<span class="page-header__counter"><?= $counter->text ?></span>
-	<? endif; ?>
-	<?= $app->renderView('Wikia\PageHeader\PageHeader', 'subtitle'); ?>
-	<hr class="page-header__separator">
+	<div class="page-header__main">
+		<?= $app->renderView( 'Wikia\PageHeader\PageHeaderController', 'categories' ); ?>
+		<h1 class="page-header__main-title"><?= !empty( $pageTitle->prefix ) ? '<span>' . $pageTitle->prefix . ':</span> ' : '' ?><?= $pageTitle->title ?></h1>
+		<?= $app->renderView( 'Wikia\PageHeader\PageHeader', 'subtitle' ); ?>
+	</div>
+	<div class="page-header__contribution">
+		<? if ( $counter->isNotEmpty() ) : ?>
+			<span class="page-header__contribution-counter"><?= $counter->text ?></span>
```

Comment:
@JamesHawking can we get rid of 'contribution' from class name? it is misleading as it is page counter, not contribution counter